lib/purescript-cst/src/Language/PureScript/Types.hs

```diff
@@ -222,8 +238,9 @@ typeToJSON annToJSON ty =
       variant "TypeApp" a (go b, go c)
     KindApp a b c ->
       variant "KindApp" a (go b, go c)
-    ForAll a b c d e ->
+    ForAll a b c d e _ ->
       case c of
+        -- TODO: Restore these in the future
         Nothing -> variant "ForAll" a (b, go d, e)
         Just k -> variant "ForAll" a (b, go k, go d, e)
```

Comment:
Could you clarify what the TODO means?

Reply:
8627ab110f70c565eabc03ad1b454dfec09b604d reverts some changes in the serialization/deserialization of types to make tests pass, since the addition of the extra field is a breaking change.